rosie/src/main/java/com/karumi/rosie/view/IRosieActivity.java

```diff
@@ -0,0 +1,5 @@
+package com.karumi.rosie.view;
+
+interface IRosieActivity {
```

Comment:
We try to give semantic naming to classes, can you rename this class to Injectable? because we can use this class on services in a near future